modules/navier_stokes/test/tests/finite_volume/fvbcs/wall_function/Re_t395.i

```diff
@@ -114,20 +119,18 @@ velocity_interp_method='rc'
   [v_advection]
     type = INSFVMomentumAdvection
     variable = v
-    advected_quantity = 'rhov'
-    vel = 'velocity'
     advected_interp_method = ${advected_interp_method}
     velocity_interp_method = ${velocity_interp_method}
-    pressure = pressure
     u = u
     v = v
-    mu = ${mu}
     rho = ${rho}
+    momentum_component = 'y'
   []
   [v_viscosity]
-    type = FVDiffusion
+    type = INSFVMomentumDiffusion
     variable = v
-    coeff = ${mu}
+    mu = ${mu}
+    momentum_component = 'y'
   []
   [v_viscosity_rans]
```

Comment:
so the symmetry BCs in that input are wrong. They use just mu instead of mu + turbulent_mu
You need the turbulent viscosity material to define the sum

Since you are regolding it do you mind fixing that?

Reply:
So the symmetry bcs are one of the few objects that should total mu instead of mu? I was looking at `mixing_length_total_viscosity.i` and noting that the symmetry BCs are the only place where we use it which feels off to me, but I don't know much about this.

Reply:
I need to double check about the WallFunctionBCs.

Reply:
I updated this such that the symmetry bcs are now using `total_viscosity`